### PR TITLE
New bookmarklet shortcode

### PR DESCRIPTION
**Pull request by punalsc**
Hi @domingobishop ,

Could you have a look at this PR. Below is a list of changes done.

* Added new bookmarklet shortcode.
* Added a new test.
* Fixed global theme test.
* Refactored code.

Thanks

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -35,6 +35,7 @@ function tna_child_styles() {
 
 //Add shortcode
 add_shortcode( 'a-z', 'a_z_shortcode' );
+add_shortcode( 'bookmarkLet', 'archive_bookmarkLet' );
```
Comment (domjngo) [inline functions.php]: Naming Conventions: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions

File: inc/functions/shortcode.php [hunk 2]
```diff
@@ -3,13 +3,19 @@
...
 	foreach ( range( 'A', 'Z' ) as $letter ) {
 		$list_items .= sprintf( "<li><a href='#%s'>%s</a></li>", strtolower( $letter ), $letter );
 	}
 	$list_items .= '<li><a href="#0-9">0-9</a></li>';
-	return sprintf($template, $list_items, $clearfix);
+	return sprintf($template, $list_items);
+}
+
+/* BookMarklet Shortcode */
+function archive_bookmarkLet($atts, $bookMarkText = null){
```
Comment (domjngo) [inline inc/functions/shortcode.php]: Naming Conventions: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions

**Review by domjngo — CHANGES_REQUESTED** [review]
See comments

File: functions.php (hunk 1 above)
Comment (punalsc) [reply functions.php]: Ok, Done

File: inc/functions/shortcode.php (hunk 2 above)
Comment (punalsc) [reply inc/functions/shortcode.php]: Ok, Done

**Review by domjngo — APPROVED** [review]

**Pull request merged**